### PR TITLE
BatchAI. Fix table output for node list and minor fixes in descriptions

### PR DESCRIPTION
**Pull request by AlexanderYukhanov**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-batchai/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+0.3.1
++++++
+* Fixed `-o table` option for `az batchai cluster node list` and `az batchai job node list` commands.
+* Fixed description and examples for `--setup-task` usage. The setup task is running under root account.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-batchai/HISTORY.rst]: Don't include changes to help content in the update history.  The first entry is fine though.

File: src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def load_arguments(self, _):
         c.argument('json_file', options_list=['--config-file', '-f'], help='A path to a json file containing cluster create parameters (json representation of azure.mgmt.batchai.models.ClusterCreateParameters).', arg_group='Advanced')
 
     with self.argument_context('batchai cluster create') as c:
-        c.argument('setup_task', help='A command line which should be executed on each compute node when it\'s got allocated or rebooted. The task is executed under a user account added into sudoers list (so, it can use `sudo`).', arg_group='Setup Task')
+        c.argument('setup_task', help='A command line which should be executed on each compute node when it\'s got allocated or rebooted. The task is executed in a bash subshell under root account.', arg_group='Setup Task')
```
Comment (tjprescott) [inline src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py]: What happens if it is run on CMD.exe then?

File: src/command_modules/azure-cli-batchai/HISTORY.rst (hunk 1 above)
Comment (AlexanderYukhanov) [reply src/command_modules/azure-cli-batchai/HISTORY.rst]: thanks, removed

File: src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py (hunk 2 above)
Comment (AlexanderYukhanov) [reply src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py]: Batch AI support Linux nodes only. The provided command will be run in bash subshell like 
bash -c cmdLine

**Pull request merged**